docs/surveys/developer-survey/README.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Gradle Build Tool - 2024 Software Developer Survey"
+description: >
+  We're doing a developer survey to prioritize our developer experience work.
+  Take the survey and get gift cards and special swag!
+---
+
+# Gradle Build Tool - Software Developer Survey
+
+The Gradle team is committed to enhancing developer experience,
+it is one of the top priorities on [Gradle Roadmap](../..roadmap/README.md/).
+Your feedback is crucial in helping us prioritize our efforts!
+
+<a class="button button--blue"
+   href="https://www.surveymonkey.com/r/7FZPXQZ"
+   target="_blank">Take the Survey
+</a>
+
+## 2024 Developer Survey
+
+![Test](./images/hero.png)
+
+The 2024 survey focuses on Gradle Build Tool developer experience, intends to get feedback on complexity of particular operations and to prioritize them.
```

Comment:
```suggestion
The 2024 survey focuses on Gradle Build Tool developer experience.  We're aiming to discover common pain points and understand what use cases are particularly frustrating.
```

Reply:
Will add it but will restore my sentence about prioritization, too